Dockerfile

```diff
@@ -51,8 +53,12 @@ RUN dnf upgrade -y \
   && ln -sfv /opt/robotframework/bin/chromium-browser /usr/lib64/chromium-browser/chromium-browser
 # FIXME: above is a workaround, as the path is ignored
 
+# Make python 3 the default python      
+RUN alternatives --install /usr/bin/python python /usr/bin/python3.7 2 \
+	&& alternatives --install /usr/bin/python python /usr/bin/python2.7 1
```

Comment:
Indentation should be spaces instead of tab like in the rest of the file. Btw what is this reference to Python 2 here? Shouldn't it also be removed?

Reply:
the command says to treat python2 lower than python3, since its still installed command should stay, it doesnt change image size just points to existing install